controllers/windowsmachine_controller.go

```diff
@@ -175,9 +175,19 @@ func isWindowsMachine(labels map[string]string) bool {
 	return false
 }
 
+// ignoreLabel checks if the ignore label is present or not
+func ignoreLabel(labels map[string]string) bool {
```

Comment:
Please don't interleave struct and helper methods. `isWindowsMachine()` is badly placed but no reason to continue that.

Reply:
Good catch; thanks for highlighting the big picture. _This is the way!_